wgsl/index.bs

```diff
@@ -4388,7 +4388,7 @@ effective-value-type.
 
 <tr><td class="nowrap">
         [=variable|var=]&lt;[=address spaces/storage=], read&gt;<br>
-        [=variable|var=]&lt;[=address spaces/storage=]&gt;<sup>5</sup>
```

Comment:
nitpick:  I don't see why the supescript should be removed.

Reply:
I removed that because this row is for read-only storage buffers which are useable everywhere. That footnote is about the restrictions on read_write storage buffers so it no longer felt relevant.